src/components/InterestSignupForm.js

```diff
@@ -33,11 +36,27 @@ class InterestSignupForm extends Component {
     render () {
         let canSubmit = (this.state.email.length > 3);
         return (
-            <div>
-                <input name="email" type="email" value={this.state.email} onChange={this.changeEmail.bind(this)} placeholder="Email"/>
-                <button type="submit" onClick={this.handleSubmit.bind(this)} disabled={!canSubmit}>Submit</button>
-                <p>{this.state.message}</p>
-            </div>);
+          <div className="con">
```

Comment:
re: ^ 

So the `Landing` component would be a good place for this markup, i.e. `<p>Sign up for updates on when applications open!</p><InterestForm/>...`